SDK6UserGuide/resources/microej.init.gradle.kts

```
@@ -0,0 +1,127 @@
+settingsEvaluated {
+
+   val userHome = System.getProperty("user.home")
+
+   allprojects {
+      repositories {
+         /* Local Repository */
+         maven {
+            name = "localRepository"
+            url = uri("${userHome}/.microej/repository")
+         }

```

Is there a specific reason not to resolve Ivy modules from local repository ?
```suggestion
         maven {
            name = "localRepository"
            url = uri("${userHome}/.microej/repository")
         }
         ivy {
            name = "localRepositoryIvy"
            url = uri("${userHome}/.microej/repository")
            patternLayout {
                artifact("[organisation]/[module]/[revision]/[artifact]-[revision](-[classifier])(.[ext])")
                ivy("[organisation]/[module]/[revision]/ivy-[revision].xml")
                setM2compatible(true)
            }
        }
```

I am not familiar with Kotlin, but I would assume it should be possible to define a construct:
```
microej {
   name = "..."
   url = uri("...")
}
```
That would be equivalent to having both `maven {}` and `ivy {}`.

---

> Maybe you meant using the Ivy local repository defined in the default SDK 5 ivy settings file: ${user.home}/.ivy2/repository?

Oh ! You're right, this is a different path.
Thing is, I used to manually drop modules in this folder when I needed to.
And that's what I've done to test the SDK6: I manually dropped "SDK 5/Ivy modules" from my app dependencies that were missing from the central/developer repositories in `~/.microej/repository`, and it failed to load them.

But it makes if you don't want to support that (one can still customize its settings anyway).

> It would add a new indirection to understand what it means. For the moment we decided to keep this file explicit.

Ok, it makes sense, I just find this file hard to understand & maintain (compared to how simple it could be).
But indeed, the indirection would make things harder for a user to understand MMM can use any Maven source.

---

ok, let's keep it like that for the moment. If we noticed more developers struggle with this, we can change it.